lrtable/src/lib/stategraph.rs

```diff
@@ -59,6 +59,15 @@ where usize: AsPrimitive<StorageT>
         StateGraph{states, edges}
     }
 
+    /// Return an iterator which produces (in order from `0..self.nonterms_len()`) all this
+    /// grammar's valid `NTIdx`s.
+    pub fn iter_stidxs(&self) -> Box<dyn Iterator<Item=StIdx>>
+    {
+        // We can use as_ safely, because we know that we're only generating integers from
+        // 0..self.states.len() which we've already checked fits within StIdxStorageT.
```

Comment:
Backquotes?

Reply:
It's a non-doc comment, so it's not markdown formatted.